test/python/WMCore_t/WMBS_t/JobSplitting_t/WMBSMergeBySize_t.py

```diff
@@ -30,7 +30,8 @@ def setUp(self):
         """
         self.testInit = TestInit(__file__)
         self.testInit.setLogging()
-        self.testInit.setDatabaseConnection()
+        self.testDB = 'unittest_%s' % self.__class__.__name__
+        self.testInit.prepareDatabase(self.testDB)
```

Comment:
A more limited comment. Why not move these two new lines into the setup() of the base class and use super() where needed? Same question with stuff that is repeatedly used in tearDown() 

I just satisfied my own curiously that `self.__class__.__name__` is the name of the derived class, even when accessed in the base class. Also, to be "nice" self.testDB should be defined in `__init__`, not here.